Fails to compile with avro-0.5.0.0

It looks as if [`avro-0.5.0.0`] has significantly changed the structure of the library, such that `language-avro` fails to compile with the following error:

```
--  While building package language-avro-0.1.2.0 using:
      /var/stackage/.stack/setup-exe-cache/x86_64-linux/Cabal-simple_mPHDZzAJ_3.0.1.0_ghc-8.8.3 --builddir=.stack-work/dist/x86_64-linux/Cabal-3.0.1.0 build lib:language-avro test:avro-parser-haskell-test --ghc-options ""
    Process exited with code: ExitFailure 1
    Logs have been written to: /var/stackage/work/unpack-dir/.stack-work/logs/language-avro-0.1.2.0.log

    Preprocessing library for language-avro-0.1.2.0..
    Building library for language-avro-0.1.2.0..
    [1 of 2] Compiling Language.Avro.Types
    
    /var/stackage/work/unpack-dir/unpacked/language-avro-0.1.2.0-b6cffdfeff894f4ffa8ada3c600d7f39bc6953f5aa00e8a048df0ab45dcd7763/src/Language/Avro/Types.hs:9:1: error:
        Could not find module ‘Data.Avro.Schema’
        Perhaps you meant
          Data.Aeson.Schema (needs flag -package-key aeson-schemas-1.0.3)
          Data.YAML.Schema (needs flag -package-key HsYAML-0.2.1.0)
        Use -v (or `:set -v` in ghci) to see a list of the files searched for.
      |
    9 | import Data.Avro.Schema
      | ^^^^^^^^^^^^^^^^^^^^^^^
    
    /var/stackage/work/unpack-dir/unpacked/language-avro-0.1.2.0-b6cffdfeff894f4ffa8ada3c600d7f39bc6953f5aa00e8a048df0ab45dcd7763/src/Language/Avro/Types.hs:10:1: error:
        Could not find module ‘Data.Avro.Types.Value’
        Use -v (or `:set -v` in ghci) to see a list of the files searched for.
       |
    10 | import Data.Avro.Types.Value
       | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

cc commercialhaskell/stackage#5262

[`avro-0.5.0.0`]: https://github.com/haskell-works/avro/releases/tag/v0.5.0.0

--- a/.github/workflows/haskell.yml
+++ b/.github/workflows/haskell.yml
@@ -5,7 +5,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        ghc: ["8.8.2"]
+        ghc: ["8.8.3"]
         cabal: ["3.0"]
     steps:
     - uses: actions/checkout@v1

--- a/language-avro.cabal
+++ b/language-avro.cabal
@@ -1,48 +1,50 @@
-name:                language-avro
-version:             0.1.2.0
-synopsis:            Language definition and parser for AVRO files.
-description:         Parser for the AVRO language specification, see README.md for more details.
-homepage:            https://github.com/kutyel/avro-parser-haskell#readme
-license:             Apache-2.0
-license-file:        LICENSE
-author:              Flavio Corpa
-maintainer:          flavio.corpa@47deg.com
-copyright:           Copyright © 2019-2020 <http://47deg.com 47 Degrees>
-category:            Network
-build-type:          Simple
-cabal-version:       >=1.10
-extra-source-files:  README.md
+name:               language-avro
+version:            0.1.3.0
+synopsis:           Language definition and parser for AVRO files.
+description:
+  Parser for the AVRO language specification, see README.md for more details.
+
+homepage:           https://github.com/kutyel/avro-parser-haskell#readme
+license:            Apache-2.0
+license-file:       LICENSE
+author:             Flavio Corpa
+maintainer:         flavio.corpa@47deg.com
+copyright:          Copyright © 2019-2020 <http://47deg.com 47 Degrees>
+category:           Network
+build-type:         Simple
+cabal-version:      >=1.10
+extra-source-files: README.md
 
 library
-  exposed-modules:     Language.Avro.Types,
-                       Language.Avro.Parser
-  build-depends:       base >=4.12 && <5
-                     , avro >=0.4.7
-                     , containers
-                     , directory
-                     , filepath
-                     , megaparsec
-                     , text
-                     , vector
-  hs-source-dirs:      src
-  default-language:    Haskell2010
+  exposed-modules:
+    Language.Avro.Parser
+    Language.Avro.Types
+
+  build-depends:
+      avro        >=0.5
+    , base        >=4.12 && <5
+    , containers
+    , directory
+    , filepath
+    , megaparsec
+    , text
+    , vector
+
+  hs-source-dirs:   src
+  default-language: Haskell2010
 
 test-suite avro-parser-haskell-test
-  type: exitcode-stdio-1.0
-  main-is:             Spec.hs
-  default-language:    Haskell2010
-  hs-source-dirs:
-      test
-  ghc-options:
-      -threaded
-      -rtsopts
-      -with-rtsopts=-N
+  type:             exitcode-stdio-1.0
+  main-is:          Spec.hs
+  default-language: Haskell2010
+  hs-source-dirs:   test
+  ghc-options:      -threaded -rtsopts -with-rtsopts=-N
   build-depends:
-      base >=4.12 && <5
-    , avro >=0.4.7
+      avro              >=0.5
+    , base              >=4.12 && <5
+    , hspec
+    , hspec-megaparsec
     , language-avro
     , megaparsec
     , text
-    , hspec
-    , hspec-megaparsec
     , vector

--- a/src/Language/Avro/Parser.hs
+++ b/src/Language/Avro/Parser.hs
@@ -22,7 +22,6 @@ where
 
 import Control.Monad (filterM)
 import Data.Avro
-import Data.Avro.Schema
 import Data.Either (partitionEithers)
 import qualified Data.Set as S
 import qualified Data.Text as T

--- a/src/Language/Avro/Types.hs
+++ b/src/Language/Avro/Types.hs
@@ -6,8 +6,7 @@ module Language.Avro.Types
   )
 where
 
-import Data.Avro.Schema
-import Data.Avro.Types.Value
+import Data.Avro
 import Data.Set
 import qualified Data.Text as T
 

--- a/stack.yaml
+++ b/stack.yaml
@@ -1,6 +1,6 @@
-resolver: lts-15.0
+resolver: lts-15.5
 packages:
 - .
 extra-deps:
 - HasBigDecimal-0.1.1
-- avro-0.4.7.0
+- avro-0.5.0.0

--- a/stack.yaml
+++ b/stack.yaml
@@ -2,5 +2,4 @@ resolver: lts-15.5
 packages:
 - .
 extra-deps:
-- HasBigDecimal-0.1.1
 - avro-0.5.0.0

--- a/test/Spec.hs
+++ b/test/Spec.hs
@@ -6,7 +6,7 @@ module Main
   )
 where
 
-import Data.Avro.Schema
+import Data.Avro
 import qualified Data.Text as T
 import Data.Vector (fromList)
 import Language.Avro.Parser
@@ -162,7 +162,7 @@ main = hspec $ do
           Nothing -- order is ignored for now...
           [ Field "name" [] Nothing Nothing String' Nothing,
             Field "age" [] Nothing Nothing Int' Nothing,
-            Field "birthday" [] Nothing Nothing (Int (Just Date)) Nothing
+            Field "birthday" [] Nothing Nothing (Int (Just Date)) Nothing
           ]
     it "should parse complex records" $
       parse parseSchema "" complexRecord

